Gemfile.lock

```diff
@@ -3,11 +3,12 @@ GEM
   specs:
     CFPropertyList (3.0.6)
       rexml
-    activesupport (7.0.4.3)
+    activesupport (6.1.7.3)
```

Comment:
I don't know what inspired me to do it, but I run `bundle install && bundle exec pod install` to update the `Podfile.lockfile` after solving the conflict on `Gutenberg/version.rb`.

I'm surprised by this change inversion though. I don't understand why it would go _back_ a version by itself.

Reply:
I think this is because [we restored `activesupport` as a dependency in the latest `release-toolkit` to avoid our _fastlane_ plugin from crashing on import](https://github.com/wordpress-mobile/release-toolkit/pull/507/files#diff-6e255cbfaf7dd800d9a43902bba94e09f67b5cf711db37e3231e987a6d857f17R27).

The revert to restore `activesupport` was made in a hurry to [stop `release-toolkit` from entirely breaking](https://github.com/wordpress-mobile/release-toolkit/pull/504#issuecomment-1642259343) so we just restored to [the same declaration that was used in the `.gemspec` before the removal](https://github.com/wordpress-mobile/release-toolkit/pull/504/files#diff-6e255cbfaf7dd800d9a43902bba94e09f67b5cf711db37e3231e987a6d857f17L27), aka `spec.add_dependency 'activesupport', '>= 6.1.7.1'`.

Technically though, I'm not sure we need a version constraint that would be that strict. Ideally we'd even find some time to quickly investigate if `PromoScreenshotsAction` still really requires the `require 'active_support/all'` (only usage of active support left in our codebase iinm) or if this is just a leftover from legacy code (or if the rare utility/helper functions from `active_support` that `PromoScreenshotsAction` uses can't be implemented differently) so we can remove the need for the dependency properly and from `release-toolkit` completely.